packages/slipshow/slipshow.0.0.32/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "A compiler from markdown to slipshow"
+description: "A longer description"
+maintainer: ["Paul-Elliot"]
+authors: ["Paul-Elliot"]
+license: "MIT"
+tags: ["slipsow" "presentation" "slideshow" "beamer"]
+homepage: "https://github.com/panglesd/slipshow"
+doc: "https://slipshow.readthedocs.io"
+bug-reports: "https://github.com/panglesd/slipshow/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "3.6"}
+  "crunch" {with-dev-setup}
+  "cmdliner"
+  "base64"
+  "bos"
+  "lwt"
+  "inotify"
+  "js_of_ocaml-compiler"
+  "js_of_ocaml-lwt"
+  "magic-mime"
+  "dream" {>= "1.0.0~alpha5"}
+  "fpath"
+  "brr" {>= "0.0.6"}
+  "ppx_blob" {>= "0.7.2"}
+  "odoc" {with-doc}
+  "ocamlformat" {with-dev-setup & = "0.26.1"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/panglesd/slipshow.git"
```

Comment:
```suggestion
conflicts: [ "ocaml-option-bytecode-only" ]
dev-repo: "git+https://github.com/panglesd/slipshow.git"
```
this will fix the `native`-only build issue on some archs